bom/runtime/pom.xml

```diff
@@ -163,6 +163,7 @@
         <proton-j.version>0.33.3</proton-j.version>
         <okhttp.version>3.14.6</okhttp.version>
         <sentry.version>1.7.30</sentry.version>
+        <datastax.java.driver.version>4.5.0</datastax.java.driver.version>
```

Comment:
Looking at other properties, I think we should rename this to `datastax-java-driver.version`.